ci/kind/test-e2e-kind.sh

```diff
@@ -131,6 +131,10 @@ case $key in
     skiplist="$2"
     shift 2
     ;;
+    --extended)
```

Comment:
did you mean to keep the changes in this file?

Reply:
Oops not intended, and will continue to add some tests, not finished yet, thanks!